fileio/ft_read_sens.m

```diff
@@ -168,20 +166,25 @@
     hdr = ft_read_header(filename, 'headerformat', fileformat, 'coordsys', coordsys, 'coilaccuracy', coilaccuracy);
     % sometimes there can also be electrode position information in the header
     if isfield(hdr, 'elec') && isfield(hdr, 'grad')
+      if isempty(senstype)
```

Comment:
oh, the default is actually applied not only for fif, but for all datasets where ft_read_header is used. So no problem here.